bluebrain/repo-patches/packages/highfive/package.py

```diff
@@ -41,17 +41,28 @@ class Highfive(CMakePackage):
     variant('mpi', default=True,  description='Support MPI')
     variant('eigen', default=False, description='Support Eigen')
     variant('xtensor', default=False, description='Support xtensor')
+    variant('page_buffer_patch', default=False, when="@2.6.2",
```

Comment:
I know that if I were to add a version, I'd probably miss updating the version requirement here - should this be `when="@2.6.2:"` instead?

Reply:
The patch only exists for `2.6.2`.